verilog/tools/kythe/scope_resolver.cc

```diff
@@ -0,0 +1,86 @@
+// Copyright 2017-2020 The Verible Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "verilog/tools/kythe/scope_resolver.h"
+
+#include <string>
+
+namespace verilog {
+namespace kythe {
+
+void Scope::AddMemberItem(const ScopeMemberItem& member_item) {
+  members_.push_back(member_item);
+}
+
+void Scope::AppendScope(const Scope& scope) {
```

Comment:
We will merge this PR, but can you cover this method in a test in a separate follow-up?

Reply:
Yes on the way. thanks for the merge.